src/components/App.tsx

```diff
@@ -1465,7 +1464,7 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     if (event[KEYS.CTRL_OR_CMD] && event.keyCode === KEYS.GRID_KEY_CODE) {
       this.toggleGridMode();
     }
-    if (event[KEYS.CTRL_OR_CMD]) {
+    if (event[KEYS.CTRL_OR_CMD] && !event.altKey && !event.shiftKey) {
```

Comment:
```suggestion
    if (event[KEYS.CTRL_OR_CMD] && !event.altKey) {
```

shift should be enabled, when we later support drag locking #1858